framework/cmd/observability/compose/conf/provisioning/dashboards/workflow-engine/engine.json

```diff
@@ -258,13 +258,54 @@
       "transparent": true,
       "type": "logs"
     },
+    {
+      "datasource": {
+        "type": "loki",
+        "uid": "P8E80F9AEF21F6940"
+      },
+      "description": "Workflow Custom Messages",
+      "gridPos": {
+        "h": 6,
+        "w": 24,
+        "x": 0,
+        "y": 22
+      },
+      "id": 61,
+      "options": {
+        "dedupStrategy": "exact",
+        "enableLogDetails": true,
+        "prettifyLogMessage": false,
+        "showCommonLabels": false,
+        "showLabels": false,
+        "showLogContextToggle": false,
+        "showTime": true,
+        "sortOrder": "Descending",
+        "wrapLogMessage": true
+      },
+      "targets": [
+        {
+          "datasource": {
+            "type": "loki"
+          },
+          "editorMode": "code",
+          "expr": "{service_name=\"unknown_service:chainlink\"} !~ \"heartbeat\"\n| json | __error__ = \"\"",
```

Comment:
we didn't manage to fix that?

Reply:
Let's update it to .*chainlink, labels can change and we don't want to reflect these changes on the dashboard